apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent, useMemo } from 'react';
+import { Outlet } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useAuthenticatedUserQuery } from '../../../../domains/auth/hooks/queries/useAuthenticatedUserQuery/useAuthenticatedUserQuery';
+import { UserAvatar } from '../UserAvatar/UserAvatar';
+
+export const Home: FunctionComponent = () => {
+  const { t } = useTranslation();
+  const { data: session } = useAuthenticatedUserQuery();
+  const user = useMemo(
+    () => ({
+      firstName: session?.user?.firstName,
+      fullName: `${session?.user?.firstName} ${session?.user?.lastName}`,
```

Comment:
Make sure we don't get a string of "undefined undefined".
```suggestion
      fullName: noNullish`${session?.user?.firstName} ${session?.user?.lastName}`,
```